src/eos-updater.c

```diff
@@ -119,13 +122,18 @@ main (gint argc, gchar *argv[])
 {
   g_autoptr(GMainLoop) loop = NULL;
   g_autoptr(OstreeRepo) repo = NULL;
-  g_auto(EosUpdaterData) data = EOS_UPDATER_DATA_INIT;
+  g_auto(EosUpdaterData) data = EOS_UPDATER_DATA_CLEARED;
```

Comment:
What was the reason to change this from `EOS_UPDATER_DATA_INIT` to `EOS_UPDATER_DATA_CLEARED`?